scripts/release/test/stage/setup/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# shellcheck disable=2029
+
+# Path(s) are relative to the root of the Jenkins workspace.
+INSTANCE=$(cat scripts/release/tmp/instance)
+
+rm -rf pkg/* && mkdir -p pkg/"$FULLVERSION"
```

Comment:
I'm pretty sure `ben-test-2.0.3` is not where we want to end up with.

Reply:
:)  Will change.